libparsec/src/config.rs

```diff
@@ -11,13 +11,18 @@ pub use libparsec_platform_device_loader::{
 };
 pub use libparsec_types::prelude::*;
 
+/// Partial client configuration.
+/// This configuration should be configured by the user before converting it to
+/// the proper configuration [`libparsec_client::ClientConfig`] by using default values as fallback.
 #[derive(Debug, Clone)]
 pub struct ClientConfig {
```

Comment:
A note for later: could we rename this to something like `PartialClientConfig` or `RawClientConfig` to make things more explicit ?

Reply:
I've created the issue #7897 for that, that would be nice to rename it :+1: